docs/dev/install.rst

```diff
@@ -129,7 +129,7 @@ running:
 
 .. code-block:: bash
 
-   python manage.py runserver
+   ./manage.py runserver localhost:8000
```

Comment:
Good point! The FxA redirect URL is `localhost` right?

We should change the `SITE_URL` section in step 4 to something like this:

```
   - ``SITE_URL`` should be set to the URL you will use to connect to your
     local development site. Some people prefer to use
     ``http://127.0.0.1:8000`` instead of ``localhost``. However, should you
     decide to change the ``SITE_URL``, you also need to
     [request](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction)
     the new ``FXA_CLIENT_ID``and ``FXA_SECRET_KEY``.
```